index.html

```diff
@@ -16,11 +16,12 @@
         <header class="header">
           <p class="logo">Welcome</p>
           <ul class="nav-list">
-            <li class="nav-item"><a href="#Welcome">Portofolio</a></li>
-            <li class="nav-item"><a href="#About">About</a></li>
-            <li class="nav-item"><a href="#Contact">Contact</a></li>
+            <li class="nav-item close-menu"><a href="#Welcome">Portofolio</a></li>
+            <li class="nav-item close-menu"><a href="#About">About</a></li>
+            <li class="nav-item close-menu"><a href="#Contact">Contact</a></li>
           </ul>
-          <nav class="fa fa-bars"></nav>
+          <nav class="fa fa-bars anime-x openMenu"></nav>
+          <i class="fa fa-close close-menu invisible"></i>
         </header>
```

Comment:
When I click on a nav link in desktop mode, the nav links disappear. Also, when I close the mobile navbar and go back to desktop mode, the navbar is not visible. Kindly fix this issue and then submit it for a re-review.